src/test/resources/com/github/checkstyle/patchfilter/MultiChangedFilesOnOnePatch.txt

```diff
@@ -0,0 +1,34 @@
+diff -u Origin/Test1.java Update/Test1.java
```

Comment:
Why this line is a part of a patch?

Reply:
when using diff to compare two directories, this line will be part of a patch

Reply:
Please show example if how it is part of patch. Terminal Command "diff -u ..." is not part of patch.

Reply:
example is https://github.com/checkstyle/patch-filters/pull/13#discussion_r439289102, I mentioned above.

Reply:
https://github.com/checkstyle/patch-filters/pull/13#discussion_r439742609 - proves that is one of possible cases.